jenkins-pipelines/features-sla-100perc-mixed-load.jenkinsfile

```diff
@@ -0,0 +1,16 @@
+#!groovy
```

Comment:
please remove all the 100perc from the file names.
change the name to features-sla-read-throughput-1to5-ratio.

Also, change the name of the test to something like that:
test_read_throughput_1to5_ratio (no need the mix) also commented about it below.